### ISSUE TITLE
stress test fails with "No node found with XXXXXX capacity"

### ISSUE THREAD

**Issue by okartau** [intel/pmem-csi#332]
This end result seems to have few different trouble paths.
One was described by @avalluri in chat:
```
pmem-csi-node/pmem-driver: Output:   /sys/dev/block/252:5/dm/uuid: fopen failed: No such file or directory
```

Another occasion contains these errors:
```
Output:scan_dev_open /dev/ndbus0region0fsdax/b01b4d7b-9e4a-11e9-ac3a-1685fe134666 failed errno 6
CreateVolume: failed to create volume: Failed to parse vgs output line:   scan_dev_open /dev/ndbus0region0fsdax/b01b4d7b-9e4a-11e9-ac3a-1685fe13466\
6 failed errno 6
```

the latter touches problem space that we cant parse all possible vgs/lvs error text-form output (also described in #218)

At least in 2nd case, there were 2 threads happening in parallel, one was in creation and another in stage.

**Comment by avalluri** [comment]
> At least in 2nd case, there were 2 threads happening in parallel, one was in creation and another in stage.

This issue should be addressed by #336

**Comment by okartau** [comment]
no such failures with latest code, closing
